cmd/poller/collector/collector.go

```diff
@@ -95,6 +94,13 @@ type AbstractCollector struct {
 	countMux *sync.Mutex // used for atomic access to collectCount
 }
 
+func (AbstractCollector) HarvestModule() plugin.ModuleInfo {
```

Comment:
looks like a placeholder to make abtractcollector interface member. is Unix a good default?

Reply:
Good catch - this is a placeholder that can be removed since the pattern is shown in pollers